### ISSUE TITLE
Feedback homework langid

### ISSUE THREAD

**Issue by stephantul** [TineDumoulin/CAI#2]
Hey Tine, 

Overall the homework looks good, although the report is missing. This makes it hard for me to interpret what your results are.

Some feedback.
In your train-test split you use a test split of .9, which is the complement of what it should be. The test split is usually .1, while the train split is .9. In an earlier version you had test split on .1, which is correct. Why did you change it?

"euclidian" did not work because you misspelled the name of the metric, it is supposed to be "euclidean".
When something doesn't work, read the documentation, this would have probably shown you that the misspelling was the source of the error.

Otherwise it looks fine. Could you write the report and submit it?

**Comment by TineDumoulin** [comment]
Hello Stephan,

I finished running the program and wrote the report. It's not very elaborate, but I think it contains the needed information.

Let me know if I have to do something else.

**Comment by stephantul** [comment]
Hey Tine, 

There's still one issue. No need to correct it, but make sure you understand. It is incorrect to train your model on a dataset which includes the test set. I realize that the homework assignment should have read "training set" instead of "dataset", this has now been corrected. In any case, you can never ever train on the same data you are testing on. 

Stéphan
